src/main/java/com/esri/samples/displayinformation/show_callout/README.md

```diff
@@ -2,7 +2,7 @@
 
 <p>Demonstrates how to show coordinates for a clicked location on an ArcGISMap in a Callout.</p>
 
-<p><img src="ShowCallout.png"/></p>
+<p><img src="ShowCallout.PNG"/></p>
```

Comment:
The Sample Viewer specifically looks for `png` and `gif` (lowercase). I don't think it will find these images if it is capitalized.